services/navigation/builtin/builtin_test.go

```diff
@@ -26,6 +26,11 @@ import (
 	"go.viam.com/test"
 )
 
+const (
```

Comment:
this doesn't need to get added - its in the same package as the previous declaration

Reply:
Tried, but am getting this when running `make lint`:

```bash
combined: services/navigation/builtin/builtin_test.go:109:3: undefined: defaultLinearVelocityMillisPerSec
```

Reply:
You named them different things, lets use the ones in this file and get rid of the ones in builtin.go

Reply:
I just realized that I wrongly assumed that "the duplicate const definitions" were happening within the same package - but they're actually happening in two separate packages. I believe they will need to remain in each of those files.